priv/perf/apps/load_testing/lib/application.ex

```diff
@@ -15,7 +17,9 @@ defmodule LoadTest.Application do
         max_connections: max_connections
       )
 
-    {:ok, self()}
+    children = [{LoadTesting.Utils.NonceTracker, %{}}, {LoadTesting.Utils.Faucet, nil}]
```

Comment:
the convention is to send an empty list if there are no arguments
```suggestion
    children = [{LoadTesting.Utils.NonceTracker, []}, {LoadTesting.Utils.Faucet, []}]
```

Reply:
Actually I left it this way just temporarily - will be passing configs in next commit.